script/mediasources.js

```diff
@@ -197,6 +199,14 @@ define('bigscreenplayer/mediasources',
         return subtitlesRequestTimeout;
       }
 
+      function getCurrentSubtitlesCdn () {
+        if (subtitlesSources.length > 0) {
+          return subtitlesSources[0].cdn;
+        }
+
+        return undefined;
```

Comment:
Is it better to just not send the `cdn` field if it's not there for some reason? Or is it a required field and therefore `undefined` is fine?

Reply:
Was following the pattern from the other functions to be honest.